Cargo.toml

```diff
@@ -20,6 +20,7 @@ members = [
     "components/properties",
     "components/segmenter",
     "components/timezone",
+    "experimental/person_names_formatter",
```

Comment:
Probably `experimental/personnames` to be more like `displaynames` and `casemapping`

Reply:
Done